ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -77,20 +66,18 @@ export function GridLayout(props: GridLayoutProps) {
         />
       )}
       <Collapse in={isOpen} unmountOnExit>
-        <Box
-          sx={{
-            display: 'grid',
-            gridTemplateColumns: `repeat(${COLUMNS}, 1fr)`,
-            gridAutoRows: {
-              xs: 24,
-              sm: 36,
-            },
-            columnGap: (theme) => theme.spacing(1),
-            rowGap: (theme) => theme.spacing(1),
-          }}
+        <ResponsiveGridLayout
+          className="layout"
+          breakpoints={{ lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0 }}
+          cols={{ lg: 24, md: 24, sm: 24, xs: 24, xxs: 2 }}
+          rowHeight={30}
```

Comment:
I feel like I'm just going to make random comments because I'm still learning the whole codebase, but do you think it's worth storing the breakpoints as constants somewhere? 

Might be fine as is

Reply:
that makes sense, but for now i'll leave it as is since i made up these breakpoints haha we can revisit when we fine tune this